From 25b2834cc974729229afcd308f127995405f85c5 Mon Sep 17 00:00:00 2001
From: Yureka <yureka@cyberchaos.dev>
Date: Sun, 12 Oct 2025 14:39:47 +0200
Subject: [PATCH] lib: bridge: avoid redefinition of in6_addr

On musl libc, which does not use the kernel definitions of in6_addr, including
the libc headers after the kernel (UAPI) headers would cause a redefinition
error. The opposite order avoids the redefinition.

Fixes: 9e89d5b94d749f37525cd8778311e1c9f28f172a
Signed-off-by: Yureka <yureka@cyberchaos.dev>
Signed-off-by: Stephen Hemminger <stephen@networkplumber.org>
---
 lib/bridge.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/bridge.c b/lib/bridge.c
index 5386aa01..104f5858 100644
--- a/lib/bridge.c
+++ b/lib/bridge.c
@@ -2,8 +2,8 @@
 
 #include <net/if.h>
 
-#include "bridge.h"
 #include "utils.h"
+#include "bridge.h"
 
 void bridge_print_vlan_flags(__u16 flags)
 {
-- 
2.51.2

